app/assets/javascripts/new_form.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function(){
```

Comment:
In future, it would be nice to get this in the format of all the other JS we are running. For now, quick fix. @nickbristow